src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/vedtaksperiode/Opphørsperiode.kt

```diff
@@ -165,16 +168,49 @@ private fun finnOpphørsperioderMellomUtbetalingsperioder(utbetalingsperioder: L
 
 fun Opphørsperiode.tilVedtaksperiodeMedBegrunnelse(
     vedtak: Vedtak,
-): VedtaksperiodeMedBegrunnelser =
-    VedtaksperiodeMedBegrunnelser(
+): VedtaksperiodeMedBegrunnelser {
+    val behandlingsÅrsak = vedtak.behandling.opprettetÅrsak
+
+    return VedtaksperiodeMedBegrunnelser(
         fom = this.periodeFom,
         tom = this.periodeTom,
         vedtak = vedtak,
         type = this.vedtaksperiodetype,
     ).also { vedtaksperiodeMedBegrunnelser ->
+        if (behandlingsÅrsak == BehandlingÅrsak.OVERGANGSORDNING_2024) {
```

Comment:
Her kunne man også vurdert en `when` for å sørge for at man må ta stilling til denne logikken om man legger til en ny årsak

Reply:
Jeg lar denne stå med tanke på at logikken som utføres i if'n heter settOvergangsordningOpphørBegrunnelser om er tett knyttet til OVERGANGSORDNING_2024